monitoring-custom-metrics/README.md

```diff
@@ -2,189 +2,51 @@
 
 ## Overview
 
-This example shows how to expose Prometheus metrics with golang from inside of a istio service mesh and monitoring that service from outside of the mesh with Prometheus.
+This example shows how to expose custom metrics to Prometheus with a golang service in Kyma.
 
-1. Configure Istio
-2. Expose a sample metrics application serving metrics on Port 8081.
-3. Access the exposed metrics in Prometheus
+1. Expose a sample application serving metrics on port 8081
+2. Access the exposed metrics in Prometheus
 
 ## Prerequisites
 
 - Kyma as the target deployment environment.
-- Istio 0.8
-  - Sidecar injection must be enabled ```istio-injection: "enabled"```
-  - Mutual TLS enabled.
+- If sidecar injection is not enabled for **default** namespace, then run the following:

```

Comment:
...``for the `default` Namespace, run the following command:``

Reply:
done